From ec47284142c4e2446aa5c435aa2784d321705257 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:43 -0400
Subject: [usb] host: uhci-platform - remove deprecated IRQF_DISABLED

Message-id: <1410887463-20674-32-git-send-email-dzickus@redhat.com>
Patchwork-id: 95106
O-Subject: [RHEL7 PATCH 031/591] usb: host: uhci-platform: remove deprecated IRQF_DISABLED
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 2b4ef839d6253e13f218f4782b1a5a595a15c747
Author: Michael Opdenacker <michael.opdenacker@free-electrons.com>
Date:   Sun Oct 6 08:47:55 2013 +0200

    usb: host: uhci-platform: remove deprecated IRQF_DISABLED

    This patch proposes to remove the use of the IRQF_DISABLED flag

    It's a NOOP since 2.6.35 and it will be removed one day.

    Signed-off-by: Michael Opdenacker <michael.opdenacker@free-electrons.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/uhci-platform.c b/drivers/usb/host/uhci-platform.c
index f1db61a..50d2f7f 100644
--- a/drivers/usb/host/uhci-platform.c
+++ b/drivers/usb/host/uhci-platform.c
@@ -105,8 +105,7 @@ static int uhci_hcd_platform_probe(struct platform_device *pdev)
 
  uhci->regs = hcd->regs;
 
- ret = usb_add_hcd(hcd, pdev->resource[1].start, IRQF_DISABLED |
-        IRQF_SHARED);
+ ret = usb_add_hcd(hcd, pdev->resource[1].start, IRQF_SHARED);
  if (ret)
   goto err_uhci;
 
-- 
1.7.1